src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  min-width: 1024px;
+}
+
+.header {
+  margin: 0;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+}
+
+.nav_panel,
+.nav_list {
+  display: flex;
+  justify-content: center;
+  text-transform: uppercase;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  display: flex;
+  padding-left: 20px;
+  line-height: 60px;
+}
+
+.nav_item:first-child {
+  padding-left: 0;
+}
+
+.link {
+  position: relative;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: black;
+}
+
+.is-active,
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  color: #00acdc;
+  margin: 0;
+  content: "";
+  height: 4px;
+  width: 37px;
+  background-color: #00acdc;
```

Comment:
Instead of setting a fixed width for the is-active::after element, use width: 100% to make it fully adapt to the width of the link.

Reply:
Done, pls check

Reply:
Move all repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)